### PR TITLE
adding services

### PR DESCRIPTION
**Pull request by RBN2208**

### CODE REVIEW

File: src/components/ServiceCard/ServiceCard.js [hunk 1]
```diff
@@ -1,38 +1,68 @@
...
+          </Button>
+        </ButtonBox>
       </Servicebox>
-      {isVisible && <CardInfo>{costs}€</CardInfo>}
+      {isVisible && <CardInfo>Preis: {costs}€</CardInfo>}
     </CardContainer>
   )
+
+  function handleClickPlus(e) {
+    e.stopPropagation()
```
Comment (marcushohlbein) [inline src/components/ServiceCard/ServiceCard.js:49]: nice job 🤩

File: src/components/ServiceCard/ServiceCard.js [hunk 2]
```diff
@@ -1,38 +1,68 @@
...
+    onPlus(costs)
+    setCounter(counter + 1)
+  }
+
+  function handleClickMinus(e) {
+    e.stopPropagation()
+    onMinus(costs)
+    setCounter(counter - 1)
+  }
```
Comment (marcushohlbein) [inline src/components/ServiceCard/ServiceCard.js:58]: you have dependency injections, you made the rules. well done 👍

**Review by marcushohlbein — COMMENTED** [review]
Very good. have some small comments in the code.

File: src/components/Button/Button.spec.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import Button from './Button'
+import '@testing-library/jest-dom'
+
+describe('Button', () => {
+  it('renders a button with a prop as backgroundcolor', () => {
+    render(<Button bgColor={{ name: 'green' }} />)
+    expect(screen.getByRole('button')).toBeInTheDocument()
+    expect(screen.getByRole('button')).toHaveStyle('background-color: green')
```
[the remark this answers is not in the document]
Comment (RBN2208) [reply src/components/Button/Button.spec.js:9]: Yes, tried it with #90ee90 and it works. When i change the hex it fails, so yeah, it works :)

File: src/components/ServiceCard/ServiceCard.js (hunk 2 above)
Comment (RBN2208) [reply src/components/ServiceCard/ServiceCard.js:58]: Thats a good point, i think i will change the counter so it cant get negative

Comment (RBN2208) [reply src/components/ServiceCard/ServiceCard.js:59]: https://github.com/RBN2208/capstone-project/pull/7/commits/d9aa604c4dbf17e8197587177c8fdd6194b07c92

Here is a link to the updatet commint

File: src/components/ServiceCard/ServiceCard.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ export default function ServiceCard({ name, costs, onPlus, onMinus }) {
         <ButtonBox>
           <span>{counter}x</span>
           <Button
+            disabled={counter === 0}
```
Comment (marcushohlbein) [inline src/components/ServiceCard/ServiceCard.js:29]: awesome :)

**Pull request merged**